internal-api/src/main/java/datadog/trace/api/rum/RumInjector.java

```diff
@@ -77,6 +94,19 @@ public byte[] getSnippet(String encoding) {
     return this.snippetCache.computeIfAbsent(encoding, this.snippetBytes);
   }
 
+  /**
+   * Gets the marker chars to inject RUM SDK after.
+   *
+   * @return The marker chars, {@code null} if RUM injection is disabled.
+   */
+  @Nullable
+  public char[] getMarker() {
```

Comment:
```suggestion
  public char[] getMarkerChars() {
```
just to make clear this returns chars whereas `getMarker(String encoding)` returns bytes

(you could also change the old method to `getMarkerBytes` to make it even clearer)